.circleci/config.yml

```diff
@@ -23,6 +24,19 @@ parameters:
     default: "spor-tf"
 
 jobs:
+  release-npm-package:
+    docker:
+      - image: circleci/node:20
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          name: Create Release Pull Request or Publish to npm
```

Comment:
Dette steget vil forsåvidt ikke opprette en PR for oss. Den vil bare publishe til npm 😔